tests/tsconfig.json

```diff
@@ -6,5 +6,9 @@
     "rootDir": "../",
     "esModuleInterop": true
   },
-  "include": ["./**/*", "../src/**/*"]
+  "include": [
+    "./**/*",
+    "../src/**/*",
+    "../tests-legacy/integration/onboarding/onboarding.selectors.ts"
```

Comment:
is this necessary? 🤔

Reply:
Noticed this too, not sure why it was added bc I didn't add it manually. I will remove it.